tests/test_suites/ShortSystemTests/test_read_cache_low_level_percentage.sh

```diff
@@ -0,0 +1,30 @@
+timeout_set 2 minutes
+
+MOUNT_EXTRA_CONFIG="sfscachemode=NEVER`
+		`|readcachemaxsizepercentage=1`
+		`|maxreadaheadrequests=0`
+		`|sfsioretries=14`
+		`|cacheexpirationtime=10000" \
+    setup_local_empty_saunafs info
+
+cd "${info[mount0]}"
+
+# Calculate total system memory in MB
+total_memory_mb=$(free --mega | awk '/^Mem:/ {print $2}')
+
+# Calculate 2% of total memory
```

Comment:
What about testing a bigger percent between `readcachemaxsizepercentage` and `FILE_SIZE`?
What if we define `required_size_mb` equals to 10% bigger than `readcachemaxsizepercentage`?

Reply:
In this test we tried to check that even if the read cache memory percentage is lower than a given file that we would like to read, it could handle the situation of no enough read cache memory by retrying the read operation  and releasing corresponding memory on the `ReadCache`, allowing the client to correctly finish the reading operation and not to freeze or not finishing the read operation. For that specific case of 10% file size bigger that the `readcachemaxsizepercentage` that you suggest, it would be necessary to increase the `sfsioretries` option in order to increase the number of times the read process releases memory from the `ReadCache` and then increase the probability of completing the read request successfully.